pinot-core/src/main/java/org/apache/pinot/core/operator/filter/predicate/EqualsPredicateEvaluatorFactory.java

```diff
@@ -106,6 +106,19 @@ public boolean applySV(int dictId) {
       return _matchingDictId == dictId;
     }
 
+    @Override
```

Comment:
Suggest adding some comments on why we override the method with the same code as the default implementation to prevent future cleanup removing the override